### DISCUSSION TITLE
bubbles.Help and tea.Model interface

### DISCUSSION THREAD

**Discussion by remogatto** [charmbracelet/bubbles#482] [Q&A] [answered]
Hello everyone,

I've noticed that `bubbles.Help` does not conform to the `tea.Model` interface. Could you clarify if there's a specific reason for this, or if I might have overlooked something? Any guidance you can provide would be greatly appreciated. Thank you in advance.

**Comment by batt0s** [comment]
Hey, I am not sure, but maybe instead of `bubbles.Help` you should use `bubbles.Help.Model`?

**Reply by remogatto** [reply]
Yes, I apologize for the confusion. My intention was to discuss the `help.Model`. However, I've noticed that it does not conform to the `tea.Model` interface due to the `View()` method's signature. Is this an intentional design choice? Thank you.

**Comment by KevM** [comment, marked as answer]
Looking at the code it seems `bubbles.Help` is component-like but does not implement `tea.Model`. The only difference is that it's View method takes a `KeyMap` to figure out which key bindings to show in the help view. The help component is meant to be encapsulated by another view.

**Reply by remogatto** [reply]
It makes sense. Thank you!
